package.json

```diff
@@ -10,7 +10,7 @@
   "author": "Jacob Johnson",
   "license": "ISC",
   "dependencies": {
-    "express": "^4.14.1"
+    "express": "^4.15.3"
```

Comment:
I like to update my packages whenever I get a chance. Express got a little behind, best to get the latest features 💃